src/main/java/dekvall/pushnotification/PushNotificationsConfig.java

```diff
@@ -3,14 +3,48 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
-@ConfigGroup("pushnotifications")
+@ConfigGroup(PushNotificationsConfig.GROUP)
```

Comment:
You can just keep this as is, i don't see it being used anywhere else

Reply:
Fair point, I just saw it done this way on the Runelite Plugin wiki page so mimicked it